azure-pipelines-PR.yml

```diff
@@ -309,31 +315,93 @@ stages:
           steps:
           - checkout: self
             clean: true
+            
+          - task: DownloadPipelineArtifact@2
+            displayName: 'Download Build Artifact'
+            inputs:
+              artifact: WindowsNoRealsig_Build
+              path: '$(Build.SourcesDirectory)/artifacts'
+              
+          - script: |
+              echo "Listing downloaded pipeline artifact"
+              dir "$(Build.SourcesDirectory)/artifacts"
+            displayName: 'List Pipeline Workspace'
 
-          - script: eng\CIBuild.cmd -compressallmetadata -buildnorealsig -testDesktop -configuration Release
+          - pwsh: eng\Build.ps1 -ci -restore -testCoreclr -configuration Release
```

Comment:
`-skipbuild` doesn't work here, because it does not restore dotnet tools.

Reply:
Why dropping `testDesktop` ?

Reply:
It's just the diff got quite scrambled. This belongs to testCoreClr job.